output_template/python/run.py

```diff
@@ -64,17 +64,15 @@ def _save_images(output_dir, filename_images):
         print("save image: {}".format(output_file_name))
 
 
-def main_test(input_image, library, config_file, max_percent_incorrect_values=0.1):
-    if not input_image or not library or not config_file:
-        print('Please check usage with --help option')
-        exit(1)
-
-    config = load_yaml(config_file)
+def _run(model, input_image, config):
+    filename, file_extension = os.path.splitext(model)
 
-    # load and initialize the generated shared library
-    nn = NNLib()
-    nn.load(library)
-    nn.init()
+    if not file_extension == '.so' or not file_extension == '.pb':
```

Comment:
When `.so` and `.pb`file are chosen, raise Exception always.

Reply:
Sorry, I mislooked I'll fix this.